src/components/alerts/Alert.vue

```diff
@@ -63,8 +65,13 @@ const attrs = useAttrs();
         <RuiIcon :name="usedIcon" size="22" />
       </div>
       <div class="space-y-1 flex-grow" :class="css.texts">
-        <div v-if="title" class="font-medium">{{ title }}</div>
-        <div class="text-body-2">{{ description }}</div>
+        <div v-if="slots.title"><slot name="title" /></div>
+        <div v-else-if="title" class="font-medium">{{ title }}</div>
+
+        <div class="text-body-2">
+          <div v-if="description">{{ description }}</div>
+          <slot />
+        </div>
```

Comment:
```suggestion
      <div v-if="slots.title || title" :class="{
          'font-medium': !!title
        }">
          <slot name="title">
            {{ title }}
          </slot>
        </div>

        <div class="text-body-2">
          <slot>
            {{ description }}
          </slot>
        </div>
```

Why not use default slot values instead?